### ISSUE TITLE
Enhancement for block domains and services

### ISSUE THREAD

**Issue by ArchiveBase** [AdguardTeam/AdGuardHome#1145]
#### Issue Details

* Version of AdGuard Home server
  *  v0.99.1
* How did you setup DNS configuration
  * System
* Operating system and version
  * Windows 10 1903

#### Steps to Reproduce
- Play any video on youtube and keep the tab opened.
- Apply custom filtering rule for `||youtube.com^ ||ytimg.com^ ||youtu.be^` or activate 'Youtube' through blocked services.
- Back to browser, ofcourse you can keep watch existing video till end, go to suggested video display on side panel and watch them etc but you may see featured videos preview broken(black) and it's only limited for thumbnails.

However i guess such feature not included to adguard home so this's not a bug with filtering or block services. 

_Additional details about some services(usually client end softwares), that not affect widely, when server identified the client network next things goes to IP level then block domain useless. in this cause even many ways for solve that but i prefer add a simple suggestion. 
Basically adguard home able collect all connected IPs history belong specific domains (include subdomains) when user block certain service then possible block that all IPs together https://github.com/AdguardTeam/AdGuardHome/issues/391_

#### Expected Behavior
As Parents expect. they want block immediately include established connections.

#### Actual Behavior
As Kids expect. It seems AG has secret deal with them😉

**Comment by copacetic82** [comment]
YouTube will stop working once Windows makes a new DNS request, and gets a reply from AGH. This usually only happens when the TTL of the DNS entry runs out (for youtube I get 5mins), if you manually do a "ipfconfig /flushdns" on Windows, reconnect the network. Even then the browser might still show cached content.

This DNS query "decision" is made by the client. To achieve immediate blocking, AGH would have to overwrite the TTL of DNS replies to e.g. 1sek, in case the user wants to block certain services.

**Comment by 1thesaurus** [comment]
@ArchiveBase You want to simplify it. How could you forget about blocking malicious domains and some people host public adguardhome servers.

**Comment by ArchiveBase** [comment]
I just preferred to suggest the feature because that is kind of important part for parents rather than explain technically 🤭

**Comment by ameshkov** [comment]
@copacetic82 even if AG rewrites the TTL, it won't help with connections that are already open.

Unfortunately, it's impossible to implement this request on the DNS-level.

It is possible to do on a proxy-level once #391 is done, but we didn't plan to implement parental control features there so I'll mark this as a low-priority feature request.

**Comment by JoeJenningsDMC** [comment]
@ameshkov, I'd like to kindly request that this feature that multiple people have asked for (Including myself) is not considered as a low priority ticket.

**Comment by ameshkov** [comment]
@JoeJenningsDMC please add the upvote reaction to the image. This is how we sort issues when we look for the most demanded

**Comment by Archive5** [comment]
> o enable and disable filtering. Unfortunately, I've noticed that even though filtering is now disabled, the domain is cached as blocked so you have to wait for the cache to clear before the domain will work again.
https://github.com/AdguardTeam/AdGuardHome/issues/1333
